aten/src/ATen/native/cuda/Embedding.cu

```diff
@@ -315,18 +316,20 @@ Tensor & embedding_renorm_cuda_(Tensor & self, const Tensor & indices,
       num_indices
     );
 
-    constexpr int num_threads = 128;
-    static_assert(num_threads % C10_WARP_SIZE == 0 &&
-                  num_threads <= cuda_utils::kCUDABlockReduceMaxThreads,
-                  "BlockReduceSum requires all warps be active");
+    int warp_size = at::cuda::warp_size();
+    int max_threads = warp_size * warp_size;
+    const int num_threads = warp_size * 2;
```

Comment:
Why this is changed from 128 to warp_size * 2?

Reply:
reverted.